### ISSUE TITLE
Face mesh

### ISSUE THREAD

**Issue by tobyclh** [google/mediapipe#3618]
<em>Please make sure that this is a [solution](https://google.github.io/mediapipe/solutions/solutions.html) issue.<em>

**System information** (Please provide as much relevant information as possible)
- Have I written custom code (as opposed to using a stock example script provided in Mediapipe):
No

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04, Android 11, iOS 14.4):
Linux python

- [MediaPipe version](https://github.com/google/mediapipe/releases):
Latest on pip 

- Solution (e.g. FaceMesh, Pose, Holistic):
FaceMesh

- Programming Language and version ( e.g. C++, Python, Java):
Python

**Describe the expected behavior:**
I want to see whether there is a way to specify the bounding box provided to the facial landmark detection graph.
Basically I want to do the facial landmark detection with known bounding box. Thank you.

**Standalone code you may have used to try to get what you need :**

If there is a problem, provide a reproducible test case that is the bare minimum necessary to generate the problem. If possible, please share a link to Colab/repo link /any notebook:

**Other info / Complete Logs :**
Include any logs or source code that would be helpful to
diagnose the problem. If including tracebacks, please include the full
traceback. Large logs and files should be attached:

**Comment by qhanson** [comment]
One way is that we reverse the tflite model to [(pytorch  face mesh)](https://github.com/thepowerfuldeez/facemesh.pytorch). Then, we need to figure out the input to this model. The facemesh used aligned face with left and right eyes, further with 25% padding.

**Comment by sureshdagooglecom** [comment]
Hi @tobyclh ,
We have similar issue https://github.com/google/mediapipe/issues/3423 and waiting for resolution , we will keep you posted once get resolution.

**Comment by tobyclh** [comment]
Hi @sureshdagooglecom 
Thanks for coming back. 
I am directly using the [TFLite version of the model](https://github.com/google/mediapipe/blob/master/mediapipe/modules/face_landmark/face_landmark_with_attention.tflite) and it does solve my problem. However, I have another question, the output of the TFLite model only has 2D output for the refined landmarks while the mediapipe python interface directly provide 3d positions. Is there any documentation that describe how should they be matched?
Thank you.

**Comment by sureshdagooglecom** [comment]
Hi @tobyclh ,
Please open another new issue for different question and if you found resolution for this issue please close the issue.
